examples/stress_tests/many_buttons.rs

```diff
@@ -96,6 +97,12 @@ fn main() {
     app.insert_resource(args).run();
 }
 
+fn set_text_colors_changed(mut colors: Query<&mut TextColor>) {
```

Comment:
This staying?

Reply:
Yeah I think it should, just in case of regressions.